doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,241 @@
+| Plutus ledger                                                                |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-ledger``                                 |
++--------------+---------------------------------------------------------------+
+
+`Plutus ledger <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-ledger>`_ is a set of transitional types that simplify the cardano-api types. 
+It is intended to be a comprehensive, easy-to-use set of types that replicate the current era of `cardano-api <https://input-output-hk.github.io/cardano-node/cardano-api/lib/Cardano-Api.html>`_. 
+It currently considers only the last era. 
+Plutus ledger contains data types and functions that complement `cardano-ledger <https://github.com/input-output-hk/cardano-ledger>`_ related to Plutus. 
+
+10. Plutus Tx constraints 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| Plutus Tx constraints                                                        |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-tx-constraints``                         |
++--------------+---------------------------------------------------------------+
+
+`Plutus-tx-constraints <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-tx-constraints>`_ contains a constraints-based API that can be used to generate on-chain validation functions and to build transactions by providing a list of constraints. 
+It is intended to support all the functionality from `Plutus ledger constraints <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-ledger-constraints>`_. 
+The main design is to be able to use the same constraints on-chain and off-chain in a Plutus application. 
+The off-chain part generates transactions based on types in `cardano-api <https://input-output-hk.github.io/cardano-node/cardano-api/lib/Cardano-Api.html>`_. 
+
+11. Plutus ledger constraints
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| Plutus ledger constraints                                                    |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-ledger-constraints``                     |
++--------------+---------------------------------------------------------------+
+
+`Plutus ledger constraints <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-ledger-constraints>`_ contains a constraints-based API that can be used to generate on-chain validation functions and to build ransactions by providing a list of constraints. 
+The main design is to be able to use the same constraints on-chain and off-chain in a plutus application. 
```

Comment:
```suggestion
The main design goal is to be able to use the same constraints on-chain and off-chain in a plutus application. 
```

Reply:
Somehow I missed this "goal" addition but now I've added it. It will be in the merged version.